pw_mgr/lib/crypto_utils.py

```diff
@@ -31,13 +31,14 @@ def check_db():
         open("passwords.txt", "w")
         print("Password DataBase created successfully!")
 
+init_key() #calling this func here instead of in main() to preempt FileNotFoundError exception
 
 """load key.key and encrypt password data"""
 def load_key():
-        file = open("key.key", "rb")
-        key = file.read()
-        file.close()
-        return key
+    file = open("key.key", "rb")
+    key = file.read()
+    file.close()
+    return key
 
 key = load_key()
```

Comment:
Why not move these into main instead?

Reply:
So the problem I was running into was that python wants to run the entire crypto_utils.py script when main.py imports crypto_utils functions. "key" and "fer" variables are global and call on the load_key function which tries to open key.key, which doesn't exist yet because main.py script hasn't called init_key yet by this point. Moving the init_key() call into crypto_utils ahead of the "key" and "fer" variables resolves the issue, but it's a temporary fix to diagnose the issue. I think the better solution might be to nest those variables into functions that can return "key" and "fer" so they don't automatically run and call load_key() as soon as the main script runs. I'll continue to work on it.

Reply:
I did try moving the load_key function and/or "fer" and "key" variables into main, but then I'd have to import Fernet and move over any other functions that use those fer/key vars. At that point, it sort of defeats the purpose of having a separate crypto_utils file. Moving init_key() just seemed like the simpler fix for the moment.